cdap-ui/app/features/workflows/workflow.less

```diff
@@ -12,4 +13,58 @@ body.state-workflows{
       }
     }
   }
+
+
+  .schedule-heading {
+    background-color: white;
+    border: 1px solid @border-color;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    font-weight: 700;
+    padding-top: 20px;
+    height: 70px;
+  }
+
+  .schedule-content {
+
+    .schedule-rows {
+
+      .schedule-row {
+        background-color: white;
+        border: 1px solid @border-color;
+        padding: 5px 0;
+        line-height: 50px;
+        cursor: pointer;
+
+        &.active {
+          background-color: #dcdce6;
+        }
+
+        p, i {
+          vertical-align: middle;
+          font-size: 15px;
+          margin-bottom: 0;
+        }
+      }
+    }
+
+    .schedule-detail {
+      background-color: #9d9fab;
+      color: white;
+      padding: 10px 0;
+
+      .row {
+        line-height: 30px;
+
+        p {
+          vertical-align: middle;
+          margin-bottom: 0;
+        }
+      }
+
+      table {
+        width: inherit;
```

Comment:
Is this necessary? We don't seem to specify width anywhere? Isn't this by default happening?